typescript/infra/src/config/gas-oracle.ts

```diff
@@ -54,7 +54,8 @@ function getLocalStorageGasOracleConfig(
   return remotes.reduce((agg, remote) => {
     let exchangeRate = getTokenExchangeRate(local, remote);
     if (!gasPrices[remote]) {
-      throw new Error(`No gas price found for chain ${remote}`);
+      // Will run into this case when adding new chains
```

Comment:
avoid throwing completely if we're running this for the first time with new chains added that don't have config values yet

Reply:
why wasn't this an issue in the past? do you think we should warn here just in case

Reply:
I probably introduced something with the recent refactor, gonna look into it